### DISCUSSION TITLE
A visual indicator on the "Albums" and "Artists" tab for albums/artists with lyrics already downloaded.

### DISCUSSION THREAD

**Discussion by azagthoth1** [tranxuanthang/lrcget#70] [Ideas]
May I suggest some sort of indicator in the "Albums" and  "Artists" tabs so we can know that there have been lyrics downloaded for them already? I know that in the "Tracks" tab, when a song already has lyrics, there's a green icon saying "Synced". I think that a similar indicator in the other two tabs would also be helpful, so we could instead focus on bulk-downloading lyrics for albums that have no lyrics yet.

**Comment by grrrrr** [comment]
If the lyrics are already downloaded (synced or instrumental),  lrcget should ignore those when you download again and only try those with missing or plain.

edit: assuming the option is turned on in settings
